demos/index.html

```diff
@@ -37,216 +37,216 @@
     </header>
     <main>
       <nav class="mdc-toolbar-fixed-adjust">
-        <ul class="catalog-list mdc-list mdc-list--two-line">
-          <li class="mdc-list-item">
+        <div class="catalog-list mdc-list mdc-list--two-line">
```

Comment:
I also wonder if this will effect page rank at all. Is the crawler smart enough to parse all this without the help of a site map or something. Can someone more enlightened chime in?

Reply:
My main concern as to why I did this was to make the entire lines clickable because the hover state will make users think the whole line is clickable, when in fact right now it's not. If we're concerned about the markup, what I can do instead is revert this and simply set the parent to `non-interactive` so that this UX issue doesn't occur.

Reply:
https://html.spec.whatwg.org/multipage/sections.html#the-nav-element

Seems like `<nav>` is enough for user agents, but no mention of anything else in the living standard.

Reply:
For now, maybe we should just add `role="list"` and `role="listitem"` and call it a day.

For the record, I totally understand why you made this change, and I _love_ the UX improvement it provides. I just want to make sure we don't accidentally break accessibility or SEO 🙂 

The change in DOM structure _might_ affect PageRank, but I'm not certain. I've generally seen `<nav>` elements contain a `<ul><li>`, which suggests it's a standard practice for some reason, but I don't know if it's for SEO or accessibility (or both).

From doing a few quick searches, I don't think our SEO can get any worse - Google Search doesn't appear to display any sitelinks for our demo site (likely because we have so many different navigation destinations), and the material.io catalog site seems to get ranked higher. So I'm not _too_ worried about SEO impact.

Once I get a Windows machine, I can definitively test the accessibility aspect in JAWS, but for now, I suspect adding the `role` attributes ought to suffice.

Reply:
When I saw this conversation unfolding I had reverted the new DOM structure, but I've put it back and added roles.